docs/make-docs

```diff
@@ -626,7 +636,7 @@ ${PODMAN} run \
   ${DOCS_IMAGE} \
   /entrypoint
 EOF
-    await_build http://localhost:3002 &
+    await_build http://localhost:3003 &
```

Comment:
This needs a review from someone with more context, possibly @jdbaldry?

Reply:
These are my changes and they are expected, thanks for double checking!

Reply:
Awesome, thanks